test/helper.rb

```diff
@@ -51,6 +51,10 @@ def strategy
       end
     end
   end
+
+  def http_site
+    strategy.client.site.sub(/^https?/, 'http')
+  end
```

Comment:
@mkdynamic http site is used in images path in several places, take a look for something like:
```
+    assert_equal "#{http_site}/321/picture", strategy.info['image']
```

I thought about adding helper to single test class, but it's used in different ones.